prover_rust/src/coordinator_client/errors.rs

```diff
@@ -0,0 +1,17 @@
+// TODO: refactor using enum
+pub type ErrorCode = i32;
+
+pub const Success: ErrorCode = 0;
+pub const InternalServerError: ErrorCode = 500;
+
+pub const ErrJWTCommonErr: ErrorCode = 50000;
+pub const ErrJWTTokenExpired: ErrorCode = 50001;
+
+pub const ErrProverStatsAPIParameterInvalidNo: ErrorCode = 10001;
```

Comment:
```suggestion
pub const ERR_PROVER_STATS_APIPARAMETER_INVALID_NO: ErrorCode = 10001;
```

Reply:
since all error code is enum now, the format is no more a problem.
the error code name is copied from go